### ISSUE TITLE
Disable country code change on keyup

### ISSUE THREAD

**Issue by sg-formsexpress** [jackocnr/intl-tel-input#739]
How can we programmatically disable the automatic changing of country code selection when typing? 

It is particularly problematic when a user is deleting an international prefix from a phone number using the backspace key as the country code is changing when they don't want it to.

So, ideally we need to disable the country code from chaning when delete or backspace are pressed.

**Comment by jackocnr** [comment]
Can you give a specific example of when this would be annoying for the user? From when the plugin is initialised, give the sequence of actions by the user e.g. if they select a country first, and if they type out their number first etc. And also which initialisation options you're using.

**Comment by sg-formsexpress** [comment]
Hello,  Thank you for the reply.   This is a request from our users.  What apparently is happening is:

1. They are loading a page where the phone number is already populated
2. The populated number includes Australia's international prefix (61) in the text box for (eg: +61430000000)
3. The user needs to delete the prefix so that the number is stored without the prefix
4. They move their cursor to just after the "1" in "+61"
6. They press "backspace" to delete the "1"
7. This changes the country to New Zealand (+64)
8. They press "backspace" again to delete the "6"
9. This changes the country to Austria (+43)
10. They press backspace again to delete the "+"
11. The country changes to blank

They want to retain the original "Australia" (+61) country selection the whole time.

**Comment by jackocnr** [comment]
I recommend leaving `nationalMode` enabled (it is by default), so users never have to deal with international dial codes. When it is enabled, if you initialise the plugin with a full international number, it will automatically format it to the national version (removing the international dial code). The idea is that in the code you only ever deal with full international numbers, but the plugin handles converting that to national numbers when displaying them to the user. Have you considered this option?

**Comment by sg-formsexpress** [comment]
Thank you for that suggestion.  I'll forward to the team for discussion and hopefully it may solve it.  I appreciate your prompt help.

**Comment by netgfx** [comment]
So I'm guessing there is no property to set to disable the autochange flag functionality without losing the nationalMode?

**Comment by jackocnr** [comment]
You are correct, there is no way to disable the autochange flag functionality

**Comment by ponasadomas** [comment]
Fighting the same issue. 

Didn't notice the issue myself. Became aware of this only after few users asked if something was broken. The problem is that they paste their phone numbers and delete their country code from right to left (leaving + sign to be deleted last). This swaps their country codes making them manually reselect their country.

**Comment by jackocnr** [comment]
Yes I can see that this behaviour is undesirable in this particular case, but I think this is an edge case - I think in the vast majority of cases when you are typing, it is useful to have the flag stay up to date with the number.

Also if you paste a number containing an international dial code, the flag will update to match it, so the UI is telling you it understands your number - so I think a lot of people at this point will leave it as it is.

So we're talking about (A) the subset of users that enter their number via copy and paste, and (B) the subset of those users who paste a number containing an international dial code, and then (C) of those people, the subset who then decide that despite the fact that the UI has shown it understands the number, that they then feel compelled to remove their international dial code.

I think for this small group of people, they will just have to reselect the right flag. And for everyone else, they will get this useful feature to auto update the flag as they type their number.

**Comment by HaniTanna** [comment]
Hello
my issue is edit time country code is missing 
eg: i added  something like this : +911253454567
 now if user will edit this form then number will display  1253454567 with country flag but on submit number will save  "1253454567"

i want input field value will be  +911253454567  so whenever resave number without changes

**Comment by cyanerd** [comment]
and some countries have the same phone code, so if I choose Kazakhstan and start typing i will autochange to Russia, so this behaviour should be optional...
